CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ CMAKE_MINIMUM_REQUIRED(VERSION 3.0 FATAL_ERROR)
 
 project(tdnf VERSION 2.1.1 LANGUAGES C)
 set(PROJECT_YEAR 2019)
+set(VERSION_STRING ${PROJECT_VERSION})
```

Comment:
@cbsiddharth Yes, was trying few things with cmake and got this issue of versioning. 
What does your second sentence mean? is there any action item pending here?

Reply:
Yes. Instead of `set(VERSION_STRING ${PROJECT_VERSION})` you can directly use `${PROJECT_VERSION}` everywhere.

There are 2 other places which refer to `${VERSION_STRING}` in tdnf which must be replaced with `${PROJECT_VERSION}`. Since you already have an open PR and identified the issue, please fix this also.

Reply:
Agree. Done. Thanks